OpenRA.Game/Settings.cs

```diff
@@ -23,24 +23,52 @@ public enum MouseScrollType { Disabled, Standard, Inverted }
 
 	public class ServerSettings
 	{
+		[Desc("Sets the server name.")]
 		public string Name = "OpenRA Game";
+
+		[Desc("Sets the internal port.")]
 		public int ListenPort = 1234;
+
+		[Desc("Sets the advertised port.")]
```

Comment:
This description isn't helpful unless the reader already knows what 'advertised' means in this context.